subsys/testsuite/cmock/CMakeLists.txt

```diff
@@ -0,0 +1,128 @@
+# Copyright (c) 2022 Swedish Embedded Consulting Group AB
+# Author: Martin Schröder <martin.schroder@swedishembedded.com>
+# SPDX-License-Identifier: Apache-2.0
+# Consulting: https://swedishembedded.com/go
```

Comment:
We can probably go without the advertising in Copyright statements.

I thought Zephyr did have a consultant directory at one point that you could sign up to be listed on.

@kestewart - do you have more information on that?